ignite/distributed/utils.py

```diff
@@ -426,6 +426,25 @@ def barrier() -> None:
     _model.barrier()
 
 
+def new_group(self, ranks: List[int]) -> Any:
+    if idist.backend() in ["nccl", "gloo", "mpi"]:
+        if isinstance(ranks, list) and all(isinstance(item, int) for item in ranks):
+            return dist.new_group(ranks=ranks)
+        else:
+            raise ValueError("Group should be list of int")
+    elif idist.backend() in ["xla-tpu"]:
+        if isinstance(ranks, list) and all(isinstance(item, int) for item in ranks):
+            return [ranks]
+        else:
+            raise ValueError("Group should be list of int")
+    elif idist.backend() == "horovod":
+        if isinstance(ranks, list) and all(isinstance(item, int) for item in ranks):
+            from horovod.common.process_sets import ProcessSet
+            return ProcessSet(ranks)
+        else:
+            raise ValueError("Group should be list of int")
```

Comment:
Please add a docstring and use `_model.new_group`. See `idist.barrier()` as an example